### PR TITLE
[What's new] backport to 47

### PR DESCRIPTION
**Pull request by tsmacdonald**
Context: waiting for a review from PDs https://metaboat.slack.com/archives/C02H619CJ8K/p1695817029550839

### CODE REVIEW

File: frontend/src/metabase/nav/components/WhatsNewNotification/WhatsNewNotification.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        value: currentVersion.tag,
+      }),
+    );
+  }, [currentVersion.tag, dispatch]);
+
+  if (!url) {
+    return null;
+  }
+  return (
+    <Paper my="lg" mx="auto" p="md" shadow="md" withBorder w={244}>
```
Comment (tsmacdonald) [inline frontend/src/metabase/nav/components/WhatsNewNotification/WhatsNewNotification.tsx:56]: @npretto this was triggering the linter. I got it in by committing with `--no-verify` but am…confused byit

**Comment by qnkhuat** [comment]
Checking the product [doc](https://www.notion.so/metabase/Provide-a-heads-up-when-Metabase-is-upgraded-with-what-s-new-44a28b55624f48e28dc3f616ce7cda0f) and it says this is for 48. Is there a thread where we decided that it'll be included in 47 ?

**Comment by tsmacdonald** [comment]
@qnkhuat yes: https://metaboat.slack.com/archives/C057T1QTB3L/p1695719945916789

**Review by ranquild — APPROVED** [review]

**Review by mazameli — APPROVED** [review]
Looked good to me. In order to get the notification to show up, though, I commented out:
```
if (!url) {
    return null;
  }
```
in `WhatsNewNotification.tsx`, which meant that of course clicking on the link didn't go anywhere. Are we sure that that link works?

**Comment by npretto** [comment]
@mazameli thanks, yes the link is tested also in the tests but i'll do a double check

**Pull request merged**